### PR TITLE
chore: update alpine image version

### PR DESCRIPTION
**Pull request by shotonoff**
## Issue being fixed or feature implemented

Downgraded alpine image version, due to BLS library with the latest alpine doesn't compile

## What was done?

Downgraded alpine image version

## How Has This Been Tested?

Built image locally

## Breaking Changes

N/A

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [ ] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by shumkov — APPROVED** [review]
As a hotfix for this version, it's ok. Let's have a task to make it work on the latest alpine afterwards.

**Comment by shumkov** [comment]
@shotonoff please fill description.

File: DOCKER/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # stage 1 Generate Tenderdash Binary
-FROM golang:1.17-alpine AS builder
+FROM golang:1.17.13-alpine3.15 AS builder
```
[the remark this answers is not in the document]
Comment (shotonoff) [reply DOCKER/Dockerfile:2]: actually, it is not necessary, 'cause the issue in the compiling bls library

**Review by shotonoff — COMMENTED** [review]
answered

**Pull request merged**